lib/components/form/location-field.js

```diff
@@ -76,6 +81,23 @@ class LocationField extends Component {
     })
   }
 
+  _geocodeSearch (text) {
```

Comment:
We've got a lot of autocomplete components floating around.  Maybe we need to abstract out some logic of storing and displaying autocomplete results so that we have less duplicate code to maintain.

Reply:
@evansiroky, what autocomplete components are you referring to besides `location-field.js`?